tests/sqllogictests/suites/stage/unload.test

```diff
@@ -81,3 +81,18 @@ select $1, $2 from @unload(file_format => 'tsv');
 1 2
 3 4
 5 6
+
+query
+copy into @unload/a_raw_path.csv from (select 1,2) file_format=(type=csv) single=true include_query_id=false use_raw_path=true detailed_output=true;
+----
+a_raw_path.csv 4 1
+
+query
+copy into @unload/a_raw_path.csv from (select 3,4) file_format=(type=csv) single=true include_query_id=false use_raw_path=true detailed_output=true;
```

Comment:
so `include_query_id=false use_raw_path=true` means override = true ?

Reply:
suppose we have a option，override = true by default， override = false not supported now， so we can omit it not it for now.

if we want  override = false by default, we`d better add it now.

Reply:
in sn, include_query_id=true require override=false, so  override = false seems better? let me add it

Reply:
done